pkg/dhclient/iface_test.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package dhclient
+
+import "testing"
+
+// TestSimple tests the cases where we should have at least one.
+func TestInterfaces(t *testing.T) {
+	l, err := Interfaces(".")
+	if err != nil {
+		t.Fatalf("Checking \".\": got %v, want nil", err)
+	}
+	t.Logf("Checking \".\": got %v", l)
```

Comment:
Check Len(l) >= 1?

Reply:
In the limit, it's hard to guarantee. If somebody runs this in a vm with only loopback interface, it will be 1.

Reply:
But I am checking len(l) == 0

Reply:
sorry I just realized you did >= 1, not > 1, yeah, did that.